hash/hashes.go

```diff
@@ -40,23 +47,46 @@ const (
 	MIMC_BW6_633
 	// MIMC_GRUMPKIN is the MiMC hash function for the Grumpkin curve.
 	MIMC_GRUMPKIN
+
 	// POSEIDON2_BLS12_377 is the Poseidon2 hash function for the BLS12-377 curve.
 	POSEIDON2_BLS12_377
+	// POSEIDON2_BLS12_381 is the Poseidon2 hash function for the BLS12-381 curve.
+	POSEIDON2_BLS12_381
+	// POSEIDON2_BN254 is the Poseidon2 hash function for the BN254 curve.
+	POSEIDON2_BN254
+	// POSEIDON2_GRUMPKIN is the Poseidon2 hash function for the Grumpkin curve.
+	POSEIDON2_GRUMPKIN
+	// POSEIDON2_BW6_761 is the Poseidon2 hash function for the BW6-761 curve.
+	POSEIDON2_BW6_761
+	// POSEIDON2_BW6_633 is the Poseidon2 hash function for the BW6-633 curve.
+	POSEIDON2_BW6_633
+	// POSEIDON2_BLS24_315 is the Poseidon2 hash function for the BLS21-315 curve.
+	POSEIDON2_BLS24_315
+	// POSEIDON2_BLS24_317 is the Poseidon2 hash function for the BLS21-317 curve.
+	POSEIDON2_BLS24_317
 
 	maxHash
 )
 
 // size of digests in bytes
 var digestSize = []uint8{
-	MIMC_BN254:          32,
-	MIMC_BLS12_381:      48,
-	MIMC_BLS12_377:      48,
-	MIMC_BW6_761:        96,
-	MIMC_BLS24_315:      48,
-	MIMC_BLS24_317:      48,
-	MIMC_BW6_633:        80,
-	MIMC_GRUMPKIN:       32,
+	MIMC_BN254:     32,
+	MIMC_BLS12_381: 48,
+	MIMC_BLS12_377: 48,
```

Comment:
I'm confused. Wouldn't the digest size be equal to `fr.Bytes`?

Reply:
Me too. I was following MIMC where it seems the digest size is `fp.Bytes`. @ThomasPiellard do you remember why it was set to `fp.Bytes`?